auth-server/src/services/auth.ts

```diff
@@ -241,6 +252,68 @@ export default class AuthService {
 
 	}
 
+	public async RequestResetPassword(email: string) {
+		const user = await User
+			.query()
+			.where('email', email)
+			.first()
+
+		if (!user) {
```

Comment:
We should not throw here. Not sure you paid attention but most website will always tell you: "ok we've checked the email you've send to us and if this user exists, you will receive an email".

Because this is a leak vector that would let an attacker know whether we have certain emails in our database.

Reply:
nice catch. fixing

Reply:
done